### ISSUE TITLE
[2.0][nightly] long_running_distributed_pytorch_pbt_failure failing

### ISSUE THREAD

**Issue by scv119** [ray-project/ray#27184]
### What happened + What you expected to happen

https://buildkite.com/ray-project/release-tests-branch/builds/832#01824449-a4ed-4043-9dd1-463fe9c8055e

### Versions / Dependencies

releases/2.0.0

### Reproduction script

N/A

### Issue Severity

High: It blocks me from completing my task.

**Comment by krfricke** [comment]
Will be closed once https://github.com/ray-project/ray/pull/27163 is merged into master and cherry-picked onto the release branch.

Run with 2.0.0 wheels: https://buildkite.com/ray-project/release-tests-pr/builds/11617#_
